src/Components/Common/Sidebar/SidebarItem.tsx

```diff
@@ -10,37 +10,39 @@ type SidebarItemProps = {
   ref?: React.Ref<HTMLAnchorElement>;
   text: string;
   icon: SidebarIcon;
+  onItemClick?: () => void;
   external?: true | undefined;
   badgeCount?: number | undefined;
   selected?: boolean | undefined;
   handleOverflow?: any;
-} & ({ to: string; do?: undefined } | { to?: string; do: () => void });
+} & ({ to?: string; do?: undefined } | { to?: string; do: () => void });
 
 type SidebarItemBaseProps = SidebarItemProps & {
   shrinked?: boolean;
-  ref: Ref<HTMLAnchorElement>;
 };
 
-const SidebarItemBase = forwardRef(
-  (
-    { shrinked, external, ...props }: SidebarItemBaseProps,
-    ref: Ref<HTMLAnchorElement>,
-  ) => {
+const SidebarItemBase = forwardRef<HTMLAnchorElement, SidebarItemBaseProps>(
+  ({ shrinked, external, ...props }, ref) => {
     const { t } = useTranslation();
     const { resetHistory } = useAppHistory();
 
     return (
       <Link
         ref={ref}
-        className={`tooltip relative ml-1 mr-2 h-full min-h-[40px] flex-1 cursor-pointer rounded-lg text-gray-900 transition-all duration-200 ease-in-out md:h-11 md:flex-none ${
+        className={`${props.to ? "to" : ""} ${props.do ? "do" : ""} tooltip relative ml-1 mr-3 h-full min-h-[40px] flex-1 cursor-pointer rounded-lg text-gray-900 transition-all duration-200 ease-in-out md:h-11 md:flex-none ${
```

Comment:
Looks like an aftermath of testing

Reply:
Nice find. Missed reverting this after testing.